plugins/workers/blurhash.worker.ts

```diff
@@ -0,0 +1,22 @@
+import { decode } from 'blurhash';
+
+/**
+ * Decodes blurhash outside the main thread, in a web worker
+ *
+ * @param {string} hash - Hash to decode.
+ * @param {number} width - Width of the decoded pixel array
+ * @param {height} height - Height of the decoded pixel array
+ * @returns {Promise} Returns the decoded pixels in the proxied response by Comlink
+ */
+export default async function getPixels(
+  hash: string,
+  width: number,
+  height: number
+): Promise<Uint8ClampedArray> {
+  try {
+    const pixels = await decode(hash, width, height);
```

Comment:
`decode` isn't async?

Reply:
We don't need this. Thought at first that comlink wasn't working because the function needed to be async, but it was something else, however, forgot about removing this 😋.

TLDR: The call to the worker needs to be awaited, the worker function does not.